datacube_stats/statistics.py

```diff
@@ -358,6 +358,63 @@ def measurements(self, input_measurements):
         ]
 
 
+class MedNdwi(Statistic):
```

Comment:
See the comment on the configuration file. I don't think this class needs to exist.